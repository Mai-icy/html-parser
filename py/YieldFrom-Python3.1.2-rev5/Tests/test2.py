#
#   Test delegation of initial next() call to subgenerator
#

def g1():
	print("Starting g1")
	yield from g2()
	print("Finishing g1")

def g2():
	print("Starting g2")
	yield 42
	print("Finishing g2")

for x in g1():
	print("Yielded", x)
