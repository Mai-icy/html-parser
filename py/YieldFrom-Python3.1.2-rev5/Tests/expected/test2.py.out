Starting g1
Starting g2
Yielded 42
Finishing g2
Finishing g1
